### PR TITLE
NL: Fix 3 fulfillment bugs

### PR DESCRIPTION
**Pull request by pradh**
This includes fixes for the following three bugs:
* https://github.com/datacommonsorg/website/issues/2474: validated by integration tests, a unit test and screenshot below
* https://github.com/datacommonsorg/website/issues/2527: validated by a new integration test
* https://github.com/datacommonsorg/website/issues/2467: see screenshots below

### Enable Temperature

image

### country in US

image

### "regions in" "places in"

image

image

### CODE REVIEW

File: server/lib/nl/detection.py [hunk 1]
```diff
@@ -133,9 +133,8 @@ class ContainedInPlaceType(str, Enum):
   # Typically corresponds to county equivalent
   EU_NUTS_3 = "EurostatNUTS3"
 
-  # Across is a generic containedInPlaceType which determines if the
-  # query is using the word "across".
-  ACROSS = "Across"
+  # Indicates that the fulfiller should guess the child-type.
```
Comment (jehangiramjad) [inline server/lib/nl/detection.py:136]: nit: while this works, it feel a little "hacky", no? "Guess" is an action and not a containedin place type. Wouldn't it be better to map to some OTHER type which then triggers the guessing action?

File: server/lib/nl/fulfiller.py [hunk 2]
```diff
@@ -102,12 +103,26 @@ def rank_charts(utterance: Utterance):
...
+  blocked_vars = set()
   while (i < len(sv_list)):
     if (sv_score[i] >= _SV_THRESHOLD):
-      ans.append(sv_list[i])
-    i = i + 1
+      var = sv_list[i]
+
+      # Check if an earlier var blocks this var.
+      if var in blocked_vars:
+        counters.info("blocked_vars", var)
```
Comment (jehangiramjad) [inline server/lib/nl/fulfiller.py:118]: nit: perhaps have a const for this string?

**Review by jehangiramjad — APPROVED** [review]
Thanks. A couple of nitty observations

File: server/lib/nl/detection.py (hunk 1 above)
Comment (pradh) [reply server/lib/nl/detection.py:136]: OTHER doesn't give any further info, but GUESS says something downstream should guess the type?  If we want it changed DEFAULT_CHILD_TYPE could be an option.

Let me know if you prefer that, and I can change in a follow on.

**Comment by pradh** [comment]
Thanks for the change!  I'll submit and handle followup comments in future PR.

**Pull request merged**